include/problem/MFEMProblem.h

```diff
@@ -167,15 +180,6 @@ class MFEMProblem : public ExternalProblem
     }
   }
 
-  mfem::Device _device;
-
   platypus::PropertyManager _properties;
```

Comment:
Would it be more appropriate for this to go on _problem_data as well?

Reply:
Yes, I think it is - moved into _problem_data